sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -494,6 +513,43 @@ public String getEntityPath() {
             });
     }
 
+    /**
+     * Sets the state of a session given its identifier.
+     *
+     * @param sessionId Identifier of session to get.
+     *
+     * @return The next expiration time for the session lock.
+     * @throws IllegalStateException if the receiver is a non-session receiver.
+     */
+    public Mono<Instant> renewSessionLock(String sessionId) {
+        if (!isSessionReceiver) {
```

Comment:
Are we going to add these API in `ServiceBusReceiverClient` . 
Also will we add integration test for this ?